book/src/global-logger.md

```diff
@@ -1,27 +1,66 @@
 # #[global_logger]
 
-*Applications* that, directly or transitively, use any of `defmt` logging macros need to define a `#[global_logger]` or include one in their dependency graph.
+> *Applications* that, directly or transitively, use any of `defmt` logging macros need to define a `#[global_logger]` or include one in their dependency graph.
+
 This is similar to how the `alloc` crate depends on a `#[global_allocator]`.
 
 The `global_logger` defines how data is moved from the *device*, where the application runs, to the host, where logs will be formatted and displayed.
 `global_logger` is transport agnostic: you can use a serial interface, serial over USB, RTT, semihosting, Ethernet, 6LoWPAN, etc. to transfer the data.
 
-The `global_logger` interface comprises two traits, `Write` and `Logger`, and one attribute, `#[global_logger]`.
+The `global_logger` interface comprises the trait `Logger` and the `#[global_logger]` attribute.
+
+## The `Logger` trait
+
+Firstly `Logger` specifies how to acquire and release a handle to a global logger, as well as how the data is put on the wire.
```

Comment:
```suggestion
`Logger` specifies how to acquire and release a handle to a global logger, as well as how the data is put on the wire.
```
I would drop this because the "Secondly" is now in a different section